Assets/Scripts/Effects/FogOfSanity.cs

```diff
@@ -94,7 +89,7 @@ private void UpdateFogRadius(int sanityLevel)
             }
             else
             {
-                _baseRadius = sanityLevel * 2;
+                _baseRadius = sanityLevel * 1.5f;
```

Comment:
Could this be a serialized field, so we can tweak it later on?

Reply:
✓